lib/rules/sinon-prefer-meaningful-assertions.js

```diff
@@ -47,53 +47,72 @@ var SINON_CHAI = [
   'threw',
 ];
 
-var PREFER_SINON_CHAI_MESSAGE = 'Use the more meaningful sinon-chai assertions.';
-var PREFER_SINON_ASSERT_MESSAGE = 'Use the more meaningful sinon.assert assertions.';
+const PREFER_SINON_CHAI_MESSAGE =
+  'Use the more meaningful sinon-chai assertions.';
+const PREFER_SINON_ASSERT_MESSAGE =
+  'Use the more meaningful sinon.assert assertions.';
 
 module.exports = {
   meta: {
     docs: {},
   },
 
-  create: function(context) {
+  create(context) {
     function getNearestFullExpression(node) {
-      while (node.parent && (node.parent.type === 'MemberExpression' || node.parent.type === 'CallExpression')) {
+      while (
+        node.parent &&
+        (node.parent.type === 'MemberExpression' ||
+          node.parent.type === 'CallExpression')
+      ) {
+        // eslint-disable-next-line no-param-reassign
         node = node.parent;
       }
 
       return node;
     }
 
     function isAssertExpression(node) {
-      return node.type === 'MemberExpression' &&
-        node.object.name === 'assert';
+      return node.type === 'MemberExpression' && node.object.name === 'assert';
     }
 
     function isRewritableSinonExpression(node, methods) {
-      if (!node) { return false; }
-      return (node.type === 'MemberExpression' && methods.indexOf(node.property.name) >= 0) ||
-        (node.type === 'CallExpression' && isRewritableSinonExpression(node.callee, methods));
+      if (!node) {
+        return false;
+      }
+      return (
+        (node.type === 'MemberExpression' &&
+          methods.indexOf(node.property.name) >= 0) ||
+        (node.type === 'CallExpression' &&
+          isRewritableSinonExpression(node.callee, methods))
+      );
     }
 
+    /* eslint-disable prettier/prettier*/
```

Comment:
Same here, `eslint --fix` with the prettier/prettier config breaks this. It outputs invalid code. Create a reproducible issue in issue in `eslint-plugin-prettier`.

Reply:
See prettier/eslint-config-prettier#31

Reply:
Space before the last `*`